source/documentation/gluster-hyperconverged/chap-Deploying_Hyperconverged.html.md

```diff
@@ -46,7 +46,71 @@ Refer [oVirt Nodes](../install-guide/chap-oVirt_Nodes) for instructions on insta
 
 ### Setting up the hyperconverged environment
 
-Steps for installing are detailed in this [blog post](/blog/2017/04/up-and-running-with-ovirt-4.1-and-gluster-storage/).
+Steps for installing are detailed in this [blog post](/blog/2018/02/up-and-running-with-ovirt-4-2-and-gluster-storage/).
+
+More details on the deployment wizard are explained below
+
+#### Host selection
+
+To ensure that GlusterFS related network traffic is segregated on the backend/storage network, the gluster bricks need to be defined using these interface. In this tab, provide the FQDN/IP addresses associated with the backend/storage network so that the bricks are correctly defined during the deployment process.
+
+Cockpit wizard uses ansible to perform the gluster deployment on the hosts. It is required to have key-based authentication setup for `root` user between the host running the wizard and the 3 addresses providedfor gluster hosts
+
+        On the host from which deployment is run
```

Comment:
You should mention, that it is a first host in a host list, not a host, where wizard is running

Reply:
Done